src/LtiAssignmentsGradesService.php

```diff
@@ -103,4 +120,39 @@ public function getGrades(LtiLineitem $lineitem)
 
         return $scores['body'];
     }
+
+    public function getLineItems()
```

Comment:
This is a new function that will correctly iterate through paginated requests and build up an array of lineItems in the course. Mostly copied from `findOrCreateLineItem`